web/crux/.env.example

```diff
@@ -13,6 +13,7 @@ JWT_SECRET=jwt-secret-token
 # install script docker pull disabled for the e2e tests
 AGENT_INSTALL_SCRIPT_DISABLE_PULL=true
 
+# Possible values: trace, debug, info, warn, error, and fatal
 LOG_LEVEL=verbose
```

Comment:
how is verbose the default then?

Reply:
Possible values: trace, debug, info, warn, error, and fatal
there is no verbose in that list

Reply:
Ahh 🦅